.eslintrc

```diff
@@ -2,5 +2,8 @@
   "extends": [
     "metalab/base",
     "metalab/node"
-  ]
+  ],
+  "rules": {
+    "import/no-unresolved": 0
```

Comment:
Nested packages don't have their `npm install` run, so they have modules that don't get resolved. This could be fixed with something like `forall packages - npm install && npm test` or similar.